critiquebrainz/frontend/templates/macros.html

```diff
@@ -220,3 +220,41 @@ <h4 style="margin-bottom:0;">{{ _('Rate this {}:'.format(entity_type_readable))
         -
     {% endif %}
 {% endmacro %}
+
+{% macro display_pagination(current_page, count, limit, page_info, parameter) %}
+  {% set total_pages = (count/limit)|round(method='ceil')|int %}
+  <div class="col-md-12 text-center">
+    <nav aria-label="Page navigation">
+      <ul class="pagination">
+
+        <li class="previous {{'disabled' if current_page <= 1 }}"><a aria-label="Previous" href="{{ url_for(page_info, page=page-1, **parameter) }}">&larr; {{ _('Previous') }}</a></li>
+        
+        <li {{ "class=active" if current_page == 1 }}><a href="{{ url_for(page_info, page=1,  **parameter) }}">{{ 1 }}</a></li>
+
+        {% if current_page > 5 %}
+          <li class="disabled"><a>{{ '...' }}</a></li>
+        {% endif %}
+
+        {% if current_page == 5 %}
+          <li><a href="{{ url_for(page_info, page=2, **parameter) }}">{{ 2 }}</a></li>
```

Comment:
I don't think this special case `current_page == 5` and `current_page == (total_page - 4)` (below) are worth it. Let's remove this two special cases. As a result, you'll probably also need to update `>` and `<` to `>=` and `<=` in the `...` cases.

Reply:
Ok, I'll change it.